aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,12 +30,22 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
-
-  private final ArrayList<String> comments = new ArrayList<String>();  
+public class DataServlet extends HttpServlet { 
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    final ArrayList<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("commentBody");
```

Comment:
Do these comments have any metadata? Are they just strings?

Reply:
As for now they are, but will add info such as timestamp and other things and for that I plan to create a Comment class.